tests/utils/constants/constants.go

```diff
@@ -0,0 +1,42 @@
+package constants
+
+import (
+	"os"
+	"path"
+)
+
+func initDIR() string {
+	currentDir, _ := os.Getwd()
+	return path.Join(currentDir, "contrib")
+}
+
+var configrationDir = initDIR()
+
+// Provider dirs' name definition
+const (
+	AWSProviderDIR   = "aws"
+	AZUREProviderDIR = "azure"
+	OCMProviderDIR   = "ocm"
```

Comment:
Please rename OCM to RHCS in all relevant places to avoid misunderstanding in future

Reply:
Fixed